HW3/HW3.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+from pyspark import StorageLevel
+from pyspark.sql import SparkSession
+
+
+def main():
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
+
```

Comment:
I really like this. I didn't think of doing this for my code but it makes sense since each person who runs it will have a different root password